File: 프로그래머스/0/120830. 양꼬치/양꼬치.js
function solution(n, k) {
    var answer = 0;
    
    if(Math.trunc(n/1)>0){
        answer = (12000*n + 2000*k) - (2000*Math.trunc(n/10))
    }
    else
        answer = (12000*n + 2000*k)
    
    return answer;
}

File: 프로그래머스/0/120830. 양꼬치/양꼬치.cpp
#include <iostream>
#include <string>
#include <cmath>

using namespace std;

int solution(int n, int k) {
    int answer = 0;
    int service = floor(n/10);
    
    if(n>=10){
        answer = (12000*n) + (2000*k) - (2000*service);
    } else {
        answer = (12000*n) + (2000*k);
    }
    
    return answer;
}